draft-ietf-masque-connect-udp-listen.md

```diff
@@ -132,9 +132,7 @@ Listener UDP Proxying Payload {
 IP Version:
 
 : The IP Version of the following IP Address field. MUST be 4 or 6.
-MAY be omitted if the latest previously transmitted Target IP Address and Port
-for the Context ID are the same as the current target IP Address and Port.
-If Omitted, the IP Address and UDP Port fields MUST also be omitted.
+It MUST be omitted Context ID is non zero.
```

Comment:
Nit: it would be easier to have a separate wire format ascii art for compressed instead of adding "MUST be omitted" to every field here